src/core/Global.ttslua

```diff
@@ -46,6 +46,27 @@ local SOURCE_REPO = 'https://raw.githubusercontent.com/chr1z93/loadable-objects/
 local library, requestObj, modMeta, notificationVisible
 local acknowledgedUpgradeVersions = {}
 
+-- language selection
+local LANGUAGE_CODES = {
```

Comment:
Should avoid correlated arrays.  Instead:
local LANGUAGES = {
  [1] = {
    code = "en",
    name = "English"
  },
...
}